pkg/capacityplanner/mocks.go

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
We have special folder for mocks. Let keep all mocks in one place.

Reply:
I want to use these mocks for capacityplanner testing. Some of these mocks are using types from capacityplanner package. If we move them to mocks package we will get cyclic import.

Reply:
Ok, got it.
However we move all mocks in separate places for avoiding cyclic imports in UTs )